### DISCUSSION TITLE
Is there a way to archive all emails older than x days

### DISCUSSION THREAD

**Discussion by MainTobias** [postalserver/postal#1548] [Help with using Postal]
I want to setup a mail server with postal which keeps entries for 30 days in sql and then archives the entries. So I guess my question is: Is there a way to make Postal use a different table every day so that I can simply take the tables instead of going through every single entry and check when it was sent/received? Or is there a different way to do this?

**Comment by willpower232** [comment]
Postal already has retention settings so it should be rotating the data effectively however you would have to create the archive process yourself which could be in a race with Postal for clearing out the database so Postal might not be the answer you need.
